src/sample_cli.c

```diff
@@ -11,7 +11,15 @@ int main(int argc, const char * argv[]) {
     for (int i = 0; i < allDevices.size; i++) {
         RazerDevice device = razerDevices[i];
         printf("%#06x\n", device.productId);
+        if (device.productId == 0x0099){
+            printf("Found Basilisk V3\n");
+            // printf("dpi: %u\n", razer_attr_read_dpi(device.usbDevice));
 
+            razer_attr_write_scroll_mode_static(device.usbDevice, "F", 3);
+            razer_attr_write_left_mode_static(device.usbDevice, "F", 3);
+            razer_attr_write_logo_mode_static(device.usbDevice, "F", 3);
+
+        }
```

Comment:
Can you revert your changes to this file? We don't really want to be adding support for specific devices on the sample cli.